src/common/constants.ts

```diff
@@ -70,4 +70,6 @@ export const RequestVariableDefinitionWithNameRegexFactory = (name: string, flag
 
 export const RequestVariableDefinitionRegex: RegExp = RequestVariableDefinitionWithNameRegexFactory("\\w+", "m");
 
+export const PromptCommentRegex = /^\s*(?:#{1,}|\/{2,})\s*@prompt\s+([^\s]+)(?:\s+(.*))?\s*$/m;
```

Comment:
Do we really need the `m` flag in the regex expression here?

Reply:
You're right. It is not required.